### PR TITLE
Add SegGPT to docs

### PR DESCRIPTION
**Pull request by andrew-healey**
This PR adds docs pages for the `autodistill_seggpt` base model.

### CODE REVIEW

File: docs/base_models/seggpt.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+## About SegGPT
+
+This Autodistill module uses a handful of pre-labelled images for improved accuracy.
+
+You will need some labeled images to use SegGPT. Don't have any labeled images? Check out [Roboflow Annotate](https://roboflow.com/annotate), a feature-rich annotation tool from which you can export data for use with Autodistill.
+
+## Quickstart
+
+```python
+from autodistill_seggpt import SegGPT,FewShotOntology
```
Comment (capjamesg) [inline docs/base_models/seggpt.md:25]: Can you add a space between `SegGPT,FewShotOntology`?

File: docs/base_models/seggpt.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+You can use [Roboflow Annotate](https://roboflow.com/annotate) to label a few images (5-10 should work fine). For your Project Type, make sure to pick Instance Segmentation--you'll be labelling with polygons.
+
+Once you've labelled your images, you can press Generate > Generate New Version. You can use all the default options--no Augmentations are necessary.
+
+Once your dataset version is generated, you can press Export > Continue.
+
+Then you'll get some download code to copy--it should look something like this:
+
+```python
+!pip install roboflow
```
Comment (capjamesg) [inline docs/base_models/seggpt.md:47]: Can you remove the `!pip install roboflow` statement? You should have this in a separate code snippet without the `!`. Users may not be using a notebook environment.

File: docs/base_models/seggpt.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+To use SegGPT with autodistill, you need to install the following dependency:
+
+
+```bash
+pip3 install autodistill-seggpt
+```
+
+## About SegGPT
+
+This Autodistill module uses a handful of pre-labelled images for improved accuracy.
```
Comment (capjamesg) [inline docs/base_models/seggpt.md:18]: Does this module require images for improved accuracy?

**Review by capjamesg — CHANGES_REQUESTED** [review]
Amazing work! Please review these comments and make the changes to both this commit and the `autodistill-seggpt` repository.

File: docs/base_models/seggpt.md (hunk 3 above)
Comment (andrew-healey) [reply docs/base_models/seggpt.md]: How do you feel about the new phrasing?

> SegGPT performs "in-context" segmentation. This means it requires a handful of pre-labelled "context" images.

File: docs/base_models/seggpt.md (hunk 2 above)
Comment (andrew-healey) [reply docs/base_models/seggpt.md:46]: I'd like to keep this as is--the default Roboflow Export code includes `!pip install roboflow`, so I think it'd be confusing to remove it from this example code snippet.

I added a comment below the code for non-notebook users--do you think this is alright?
> Note: if you are not using a notebook environment, you should remove `!pip install roboflow` from your code, and run `pip install roboflow` in your terminal instead.

**Review by capjamesg — APPROVED** [review]
Thank you for making the requested changes. This is ready to deploy.

**Comment by capjamesg** [comment]
@andrew-healey Can you also add SegGPT to the model compatibility matrix following these instructions:

https://github.com/autodistill/autodistill/tree/main/automation

Those instructions do not update the compatibility matrix in the docs homepage. You'll need to copy it manually. (Do not copy the full README into the `docs/index.md` folder because there are a few changes between the documents IIRC).

**Comment by capjamesg** [comment]
@andrew-healey I'll do this because there is another model we need to add 😅

**Pull request merged**